### DISCUSSION TITLE
vim mode yank to system clipboard

### DISCUSSION THREAD

**Discussion by mskry** [lukehaas/RunJS#641] [Q&A]
In vim mode the only way to copy code is from the contextual menu, is it possible to yank the code into a system clipboard ? (Linux)

**Comment by lukehaas** [comment]
I'll take a look into this and see if it's possible. RunJS relies on Codemirror and the Codemirror Vim plugin for Vim mode, so it will depend on whether this behaviour is supported there.

**Reply by mskry** [reply]
Might be here https://github.com/kometenstaub/codemirror-vim/commit/e4a7453d68d15689d8d8be49cb5b22e238e55802
But I could't get it working (set register to '+') on the web https://codemirror.net/5/demo/vim.html using the default binding 
`{ keys: '"<character>', type: 'action', action: 'setRegister' }`
